src/seed.ts

```diff
@@ -0,0 +1,44 @@
+import { faker } from "@faker-js/faker";
+import { prisma } from "./setup-db";
+import { hashPassword } from "./graphql/resolvers";
+import { setupDatabase } from "./setup-db";
+
+interface userSeedInfo {
+  name: string;
+  email: string;
+  password: string;
+  birthDate: string;
+}
+function createRandomUser(): userSeedInfo {
+  return {
+    password: faker.internet.password(),
+    email: faker.internet.email(),
+    name: faker.internet.userName(),
+    birthDate: faker.date
+      .birthdate()
+      .toLocaleDateString("en-GB", { day: "2-digit", month: "2-digit", year: "numeric" })
+      .replace(/\//g, "-")
+  };
+}
+
+const USERS: userSeedInfo[] = faker.helpers.multiple(createRandomUser, {
+  count: 5
+});
+
+async function seed() {
+  console.log("Seeding database...");
+  for (const user of USERS) {
+    const hashedPassword = await hashPassword(user.password);
+    await prisma.user.create({
+      data: {
+        name: user.name,
+        email: user.email,
+        password: hashedPassword,
+        birthDate: user.birthDate
+      }
+    });
```

Comment:
Aqui se vc for criar 50 users, ele vai fazer 50 queries pro banco. Existe o `createMany` no Prisma que te permite fazer essa criação fazendo uma query só pro banco. Num cenário de projeto real, consumiria bem menos recursos. Acredito que até msm nesse cenário local o script rode mais rápido tbm.